pytest_reportportal/rp_logging.py

```diff
@@ -16,13 +16,26 @@ def _log(self, level, msg, args,
         Low-level logging routine which creates a LogRecord and then calls
         all the handlers of this logger to handle the record.
         """
+
         sinfo = None
+        p_version_flag = False
```

Comment:
As this package already uses six, I would recommend to use `six.PY2` or `six.PY3` instead of own check. There is a plan to get rid of `six` due to future refactoring, but this might happen `when sun becomes red giant` :-)